python-backend/app/api/variances/resources/variance_resource.py

```diff
@@ -18,22 +19,47 @@ class VarianceResource(Resource, UserMixin, ErrorMixin):
         description='Get a list of variances.',
         params={
             'page': f'The page number of paginated records to return. Default: {PAGE_DEFAULT}',
-            'per_page': f'The number of records to return per page. Default: {PER_PAGE_DEFAULT}'
+            'per_page': f'The number of records to return per page. Default: {PER_PAGE_DEFAULT}',
+            'variance_application_status_code':
+            'Comma-separated list of code statuses to include in results. Default: All status codes.',
```

Comment:
is there no point to using an f-string when we don't add a {variable} ?

Reply:
That's correct 😄 the `f` allows for string interpolation, like backticks in JS

Reply:
cool